Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.centos.org/centos/centos:7
+
+RUN useradd 3scale
```

Comment:
Question: do we later run the app as this user?

Reply:
Not necessarily